src/components/Search/SearchDropdown.tsx

```diff
@@ -17,19 +17,19 @@ import { fetchRecentRevisions } from '../../thunks/searchThunk';
 import { InputType, Repository } from '../../types/state';
 
 interface SearchDropdownProps {
-  isEditable?: boolean;
+  compact: boolean;
   selectLabel: string;
   tooltipText: string;
   searchType: InputType;
 }
 
 //handle in progress repos here if necessary
 function SearchDropdown({
-  isEditable,
+  compact,
   selectLabel,
   searchType,
 }: SearchDropdownProps) {
-  const size = isEditable == true ? 'small' : undefined;
+  const size = compact ? 'small' : undefined;
```

Comment:
Ok, I see why.

Reply:
I should have provided an explanation about that in the PR.

I try to think at the component level: what is the component doing, and what does it need as parameter to drive its behavior? In this case, this property controls how it is displayed, and its compacity. That's I renamed it to `compact`. The fact that the form is editable makes no sense at this specific component level.

Reply:
Of course this doesn't change the logic at all! But this is the idea of splitting the responsibilities between the components (and the props).